lib/core/src/Cardano/Wallet/Api.hs

```diff
@@ -827,6 +832,20 @@ type GetCurrentSMASHHealth = "smash"
     :> QueryParam "url" (ApiT SmashServer)
     :> Get '[JSON] ApiHealthCheck
 
+{-------------------------------------------------------------------------------
+                                  Tokens
+
+  See also: https://input-output-hk.github.io/cardano-wallet/api/edge/#tag/Forge
+-------------------------------------------------------------------------------}
+
+type Tokens n = ForgeToken n
+
+type ForgeToken n = "wallets"
+    :> Capture "walletId" (ApiT WalletId)
+    :> "forge"
+    :> ReqBody '[JSON] (ForgeTokenDataT n)
+    :> PostAccepted '[JSON] (ApiTransactionT n)
```

Comment:
Note: I think that the latest terminology we want to adopt across tools is "mint / burn" because "forge" has been deemed confusing :shrug: